### PR TITLE
Cart Context

### PR DESCRIPTION
**Pull request by tomekwoojcik**

### CODE REVIEW

File: src/api/api-cart/api-add-cart.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+export const apiAddCart = async (
+  url: string,
+  product: ProductObjectInCartModel
+): Promise<CartModel | undefined> => {
+  const apiUrl: string = url + apiCartPath;
+  try {
+    const response = await axios.post<CartModel>(apiUrl, null, {
+      params: product,
+      //TODO wynieść do state badz config
+      //TODO .env
```
Comment (liftoff-project) [inline src/api/api-cart/api-add-cart.ts]: Is there anything else to be done here?

File: src/component/pages/main-page/main-page.tsx [hunk 2]
```diff
@@ -15,7 +16,9 @@ export const MainPage: FC = () => {
     <ThemeProvider theme={theme}>
       <Grid className="container" container>
         <Grid item>
-          <Navbar categories={category} />
+          <CartProvider>
```
Comment (liftoff-project) [inline src/component/pages/main-page/main-page.tsx:19]: Is CartProvider applied only to the Navbar?

File: src/context/cart-context.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+export interface PropsContextModel {
+  children: JSX.Element | JSX.Element[];
+}
+
+const CartContext = createContext({} as CartContextInterface);
+
+const CartProvider = ({ children }: PropsContextModel) => {
+  const [cart, setCart] = useState<CartModel | undefined>(undefined);
+  useEffect(() => {
+    const getChartData = async () => {
```
Comment (liftoff-project) [inline src/context/cart-context.tsx]: rename: getCartData

**Review by liftoff-project — APPROVED** [review]

File: src/component/pages/main-page/main-page.tsx (hunk 2 above)
Comment (tomekwoojcik) [reply src/component/pages/main-page/main-page.tsx:19]: Cart Provider is for the entire project only for testing purposes, it is used here.

**Pull request merged**